### DISCUSSION TITLE
Windows warning regarding trojan

### DISCUSSION THREAD

**Discussion by Al-kappa** [Rem0o/FanControl.Releases#1300] [Q&A] [answered]
Hey there, when it comes to software im kinda lost.
Today i started my PC and had a windows warning pop up right after i updated Fan Control to V137.
Its located in the Fan control folder and found 2 possibly harmful things.
Thats what its called: trojan:Script/Wacatac.B!ml
should i get rid of the tool for a while? o.O

**Comment by andrewchemerys** [comment]
Same warning!

**Comment by alistairgd** [comment]
Same here, I got the fan control popup notification for a new update, clicked okay, and MS Defender immediately warned of Trojan:Win32/Wacatac.H!ml  file: \FanControl\Updater.exe

**Comment by Mielie6005** [comment]
I have resolved this issue on my pc. I just copied my fan config to a different location and uninstalled all fan control files, reinstalled it, and loaded the fan config. The update was already installed and functioning like normal

**Comment by evajust** [comment]
I also got this when trying to update. I just had Windows remove the update file and I'll wait to if anything is said in this thread.

**Comment by Rem0o** [comment, marked as answer]
See https://github.com/Rem0o/FanControl.Releases/issues/1297
